PowerUp/app/src/main/java/powerup/systers/com/SelectFeaturesActivity.java

```diff
@@ -538,63 +536,65 @@ public void onClick(View v) {
             public void onClick(View v) {
                 getmDbHandler().open();
                 if (value.equalsIgnoreCase(getResources().getString(R.string.cloth))) {
-                    if(SessionHistory.totalPoints < getmDbHandler().getPointsClothes(cloth))
-                        Toast.makeText(SelectFeaturesActivity.this, R.string.points_check, Toast.LENGTH_SHORT).show();
-                    else{
+                    if(SessionHistory.totalPoints < getmDbHandler().getPointsClothes(cloth)) {
+                        Snackbar bar = Snackbar.make(v, "Sorry, you did not have enough points for that purchas", Snackbar.LENGTH_LONG);
+                        bar.show();
+                    }else{
                         getmDbHandler().setAvatarCloth(cloth);
                         getmDbHandler().setPurchasedClothes(cloth);
                         SessionHistory.totalPoints = SessionHistory.totalPoints - getmDbHandler().getPointsClothes(cloth);
                     }
                 } else if (value.equalsIgnoreCase(getResources().getString(R.string.hair))) {
-                    if(SessionHistory.totalPoints < getmDbHandler().getPointsHair(hair))
-                        Toast.makeText(SelectFeaturesActivity.this, R.string.points_check, Toast.LENGTH_SHORT).show();
-                    else{
+                    if(SessionHistory.totalPoints < getmDbHandler().getPointsHair(hair)) {
+                        Snackbar bar = Snackbar.make(v, "Sorry, you did not have enough points for that purchase", Snackbar.LENGTH_LONG);
+                        bar.show();
+                    }else{
                         getmDbHandler().setAvatarHair(hair);
                         getmDbHandler().setPurchasedHair(hair);
                         SessionHistory.totalPoints = SessionHistory.totalPoints - getmDbHandler().getPointsClothes(cloth);
                     }
                 } else if (value.equalsIgnoreCase(getResources().getString(R.string.accessory))) {
                     if (hatPurchased != 0){
-                        if(SessionHistory.totalPoints < getmDbHandler().getPointsAccessories(hatPurchased))
-                            Toast.makeText(SelectFeaturesActivity.this, R.string.points_check, Toast.LENGTH_SHORT).show();
-                        else{
+                        if(SessionHistory.totalPoints < getmDbHandler().getPointsAccessories(hatPurchased)) {
+                            Snackbar bar = Snackbar.make(v, "Sorry, you did not have enough points for that purchase", Snackbar.LENGTH_LONG);
```

Comment:
Same. Also give a more meaningful name to Snackbar instance, notEnoughtPointsBar for example.